### ISSUE TITLE
The time of building the graph is amost half of an hour

### ISSUE THREAD

**Issue by MarkAlive** [NeroLoh/darts-tensorflow#5]
Hi @NeroLoh, I found that it takes amost half of an hour before it runs to sess.run. While in [enas](https://github.com/melodyguan/enas), it costs relatively less time. Do you have the same situation?

**Comment by NeroLoh** [comment]
Comparing to ENAS, the DARTS takes about 1+4 times of gradient compute(The follower and the leader update) and should hold the copy of the training network (the "unrolled model"), It may be reseanable to takes much more of time for computing.
Howerer, It still cost a lot more of time than the pytorch version of DARTS,and I think it may caused by that the tensorflow framework itself are not so well to deal with complicate gradients computation, or may be I just didn't find the effcient way to handle it. 
I think using tensorbord  on a shrinked network to see witch part are costly and then optimizing it targetly may be useful. However, I recently may have little time to consentrate on this. If you have any idea, please contact me, and I will try it whenever I am free.

**Comment by MarkAlive** [comment]
I was wrong. This was caused by my slow tf-eager version code.
